core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.html

```diff
@@ -890,10 +890,13 @@ <h1 class="stat-value"><[$ctrl.subscriptionsList.length]></h1>
 
   .oppia-learner-dashboard-intro-button {
     background-color: #015c53;
+    background-color: #00645c;
```

Comment:
Why the  `background-color` twice here?

Reply:
#015c53 is not required and removed. Thanks for pointing out.